frontend/src/components/invite-form.ts

```diff
@@ -79,11 +81,23 @@ export class InviteForm extends LiteElement {
             )}
           </sl-select>
         </div>
+        <div class="mb-5">
+          <sl-select
+            label=${msg("Role")}
+            value=${defaultUserRole}
+            name="inviteRole"
+          >
+            <sl-option value=${AccessCode.owner}>${"Admin"}</sl-option>
+            <sl-option value=${AccessCode.crawler}>${"Crawler"}</sl-option>
+            <sl-option value=${AccessCode.viewer}>${"Viewer"}</sl-option>
+          </sl-select>
+        </div>
+
         <div class="mb-5">
           <sl-input
             id="inviteEmail"
             name="inviteEmail"
-            type="email"
+            type="text"
```

Comment:
[This setting enables the dedicated email keyboard on touch devices](https://shoelace.style/components/input#input-types).  Is there a reason this was switched to `type="text"`??